libsrc/occ/python_occ_shapes.cpp

```diff
@@ -744,6 +753,14 @@ DLL_HEADER void ExportNgOCCShapes(py::module &m)
              {
              case TopAbs_FACE:
                BRepGProp::SurfaceProperties (shape, props); break;
+             case TopAbs_SHELL:
+               BRepGProp::SurfaceProperties (shape, props); break;
+	     case TopAbs_SOLID:
+               BRepGProp::VolumeProperties (shape, props); break;
+	     case TopAbs_COMPOUND:
+               BRepGProp::VolumeProperties (shape, props); break;
+	     case TopAbs_COMPSOLID:
+               BRepGProp::VolumeProperties (shape, props); break;
```

Comment:
A little denser syntax:
```suggestion
             case TopAbs_FACE:
             case TopAbs_SHELL:
               BRepGProp::SurfaceProperties (shape, props); break;
	     case TopAbs_SOLID:
	     case TopAbs_COMPOUND:
	     case TopAbs_COMPSOLID:
               BRepGProp::VolumeProperties (shape, props); break;
```

Reply:
Done as well, thanks for the hint.